mobile/Screens/HomeScreen.jsx

```diff
@@ -47,9 +48,108 @@ const styles = StyleSheet.create({
 });
 
 export default function HomeScreen() {
+  const [allSaved, setAllSaved] = useState('');
```

Comment:
If possible, let's migrate this to a new screen! The designers have decided to move saved items from Home to a separate screen